src/aave-v3/SupplyVaultBase.sol

```diff
@@ -30,7 +29,6 @@ abstract contract SupplyVaultBase is ERC4626UpgradeableSafe, OwnableUpgradeable
     /// STORAGE ///
 
     IMorpho public immutable morpho; // The main Morpho contract.
-    IPool public immutable pool; // The AaveV3 pool.
 
     address public poolToken; // The pool token corresponding to the market to supply to through this vault.
```

Comment:
Why this address has not been set immutable ?

Reply:
because it's not at initialisation

Reply:
I tough that it was possible to set it at construction time in this case.

Reply:
I dont understand, why is it not set at initialization ?

Reply:
I think that he meant that it is set at initialization, that is why we can put it as immutable